meta-resin-common/recipes-core/packagegroups/packagegroup-resin.bb

```diff
@@ -22,6 +22,5 @@ RDEPENDS_${PN} += " \
     resin-state-reset \
     resinhup \
     aufs-util \
-    ${@bb.utils.contains('RESIN_CONNECTABLE', '1', 'resin-connectable', '', d)} \
     ${@bb.utils.contains('RESIN_CONNECTABLE', '1', 'resin-provisioner', '', d)} \
```

Comment:
I got the raised issue. That is something I will fix in the the next push. For now the PR is postponed until we clarify some aspects of the way this is shaped.